### PR TITLE
feat: Added management command to update LearnerPathwayProgress

### PR DESCRIPTION
**Pull request by zamanafzal**
**Description:** Update the existing learner pathway progress data 

**JIRA:** Link to JIRA ticket [ENT-7259](https://2u-internal.atlassian.net/browse/ENT-7259)

**Dependencies:** dependencies on other outstanding PRs, issues, etc. 

**Merge deadline:** List merge deadline (if any)

**Installation instructions:** List any non-trivial installation 
instructions.

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [ ] tag reviewer 
- [ ] tag reviewer 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.

### CODE REVIEW

File: learner_pathway_progress/management/commands/update_existing_pathways_progress.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    Command to update learner progress in pathway.
+
+    Update pathway progress for all enterprise learners who have any course enrollments.
+
+    Examples:
+
+        ./manage.py lms update_existing_pathways_progress
+    """
+    help = dedent(__doc__)
+
```
Comment (sameenfatima78) [inline learner_pathway_progress/management/commands/update_existing_pathways_progress.py:22]: I think this command can be very helpful in the future if we want to update pathway progress for a single enterprise learner. Do you think if that would be possible to update progress of one specific learner through this command?

**Review by sameenfatima78 — APPROVED** [review]
Just one comment, otherwise LGTM

File: learner_pathway_progress/management/commands/update_existing_pathways_progress.py (hunk 1 above)
Comment (zamanafzal) [reply learner_pathway_progress/management/commands/update_existing_pathways_progress.py:22]: Do you have any case in mind where we need to update the pathway progress for a single learner? Yes, it'll be possible to update the pathway progress for one learner by making a small change in the case.

Comment (sameenfatima78) [reply learner_pathway_progress/management/commands/update_existing_pathways_progress.py:22]: Like for instance if a bug shows up where a pathway progress for one specific learner isn't reflecting correctly on learner portal, we can run this management command to update it. But if we are planning to run this command periodically then we can just ignore what I said :)

Comment (zamanafzal) [reply learner_pathway_progress/management/commands/update_existing_pathways_progress.py:22]: I think that bug shouldn't come in the first place as we emit the signals and change the pathway progress of the learner when he/she meets certain criteria.
This command will run only once to fix the existing data where the signal was calling before the data was saved and cause the issue. 
If we need to run it in future for a single learner, we can pass the learner id to this command with a small change in the code and it'll work.

**Review by muhammad-ammar — APPROVED** [review]

**Review by bilaltahir21 — APPROVED** [review]
LGTM 🚢

**Pull request merged**